src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -32,12 +32,18 @@ const {
 } = require('./../../../../utilities/stringHelpers');
 
 const officePage = require('./officePage.graphql');
-/**
- * Queries for all of the pages out of Drupal
- * To execute, run this query at http://staging.va.agile6.com/graphql/explorer.
- */
-module.exports = `
 
+let regString = '';
+queryParamToBeChanged.forEach(param => {
+  regString += `${param}|`;
+});
+
+const regex = new RegExp(`${regString}`, 'g');
+
+const buildQuery = ({ useTomeSync }) => {
```

Comment:
I'm not opposed to adding a JSDoc, but a lot of this refactoring feels too temporary to warrant spending a bunch of effort on it.  Once we get all the transformers in place and have tome-sync installed on the CMS, we won't need `useTomeSync` as an option - it will just be the default and a lot of this stuff will go away.